concordium-consensus/src/Concordium/Skov/Update.hs

```diff
@@ -685,7 +685,7 @@ transactionVerificationResultToUpdateResult (TV.NotOk TV.CredentialDeploymentInv
 transactionVerificationResultToUpdateResult (TV.NotOk (TV.ChainUpdateSequenceNumberTooOld _)) = ResultChainUpdateSequenceNumberTooOld
 transactionVerificationResultToUpdateResult (TV.NotOk TV.ChainUpdateEffectiveTimeBeforeTimeout) = ResultChainUpdateInvalidEffectiveTime
 transactionVerificationResultToUpdateResult (TV.NotOk TV.CredentialDeploymentExpired) = ResultCredentialDeploymentExpired
-transactionVerificationResultToUpdateResult (TV.NotOk TV.NormalTransactionDepositInsufficient) = ResultTooLowEnergy
+transactionVerificationResultToUpdateResult (TV.NotOk TV.NormalTransactionDepositInsufficient) = ResultInsufficientFunds
```

Comment:
I don't think this is correct. This error is when the NRG amount is less than the minimal one based on transaction size https://github.com/Concordium/concordium-node/blob/1b088cb37b76f329975f73fab9523333e3fb38bc/concordium-consensus/src/Concordium/TransactionVerification.hs#L273